packages/datagrid/src/components/dataset-serializer/index.js

```diff
@@ -0,0 +1,8 @@
+import { getColumnDefs, getRowData, getPinnedColumnDefs, getValueByCell } from './utils';
```

Comment:
you will kill glorieux here: remove utils.
two options:
*  get each one of them a file `getColumnDefs.js``
* no folder and all in one file `dataset-serializer.js`

By the way why this dataset ? I don't get the name here.